common/source/docs/common-paramosd.rst

```diff
@@ -0,0 +1,134 @@
+    :width: 450px
+
+  
+Parameter slot scroll-selection
+===============================
+
+Holding roll right for > 3 seconds on a parameter's value will highlight both the selected parameter and its value (blink). This allows 
+to scroll through the full list of parameters and place another paramater into that respective slot using pitch up and down stick input. 
+Move left to exit parameter selection and got back to standard screen navigation.
+
+.. note::
+
+   OSD based parameter adjustment is designed to allow basic adjustments without any aditional data link or GCS setup required. However, 
+   some paramaters with wide value ranges might not be ideal for adjustment using parameter OSD.
+
+
+Parameter value and range mnemonics
+===================================
+
+OSD based parameter screens use mnemonics to account for complex functional parameter values and value ranges. This allows estimate useful 
+incrementation steps for individual paramater value ranges, as well as to display human-readable abbreviations for: 
+
+- flightmodes
+
+- servo functions
+
+- serial protocols
+
+- RC aux options
+
+- failsave actions
+
+.. note::
+
+   This feature my have limitations on certain parameters with exceptionally wide value ranges and adaptive incrementation steps. Selecting a 
+   different parameter using scroll selection will reset the type mnemonics to "none".
+
+
+Populating a screen's parameter slots using maxproxy's OSD module
+=================================================================
+ 
+OSD based parameter setting includes a `mavproxy module <https://ardupilot.org/mavproxy/>`__ to populate each screen's parameter slots.
+Connect your flight controller and load the OSD module by typing :code:`module load osd`
+
+Use the following syntax to set and show a parameter, its type, minimum, maximum and increments:
+
+:code:`param-set <screen> <index> <name> (<type> | <min> <max> <increment>)` 
+
+:code:`param-show <screen> <index>` 
+
+where 
+
+:code:`screen` is the desired parameter screen number (5 or 6)
+
+:code:`index`  is the desired parameter slot (1 - 9)
+
+:code:`NAME` is the desired parameter's full name (case sensitive)
+
+:code:`type` allows to set mnemonics to 
+
+- :code:`FLIGHT_MODE`
+- :code:`SERIAL_PROTOCOL`
+- :code:`SERVO_FUNCTION`
+- :code:`AUX_FUNCTION` (for RC options)
+- :code:`FAILSAFE_ACTION` (plane: battery low and critical failsafe actions; copter: failsafe options)
+- :code:`FAILSAFE_ACTION_1` (plane: short failsafe actions; copter: throttle and GCS failsafe options)
+- :code:`FAILSAFE_ACTION_2` (plane: long failsafe actions; copter: battery failsafe actions)
+- :code:`NONE`
+
+:code:`MIN` allows to set a lower limit to the setteable range
+
+:code:`MAX` allows to limit the settable range's maximum
+
+:code:`INCREMENT` allows to use specific increments if required
+
+Examples
+--------
+
+To set OSD5 parameter1 to TRIM_THROTTLE, type :code:`osd param-show 5 1 TRIM_THROTTLE`
```

Comment:
param-show?

Reply:
yes, this is correct.
we might talk to @andyp1per about the syntax. it's not super well aligned with common param handling syntax and type commands are case-sensitive, while non-osd param commands are not... but then, imho the final goal is to add GUI-type setting support in MP

Reply:
I did want to fix the case sensitivity thing.
I'm open to other syntax suggestions, although what we have is somewhat consistent with parameter syntax. Note that we can't have:
```osd param show ....```
Or at least its a pain to implement, I would prefer just a single command name, and as @vierfuffzig ideally we want MP to implement this.

Reply:
@andyp1per just brainstorming disregarding the pita to implement;
- rename module to `paramosd` as `osd` is somewhat unspecific regarding the various osd options we have now
- make syntax `paramosd set` and `paramosd show` to align with module `param`